content/pt-br/docs/reference/glossary/approver.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Aprovador
+id: approver
+date: 2018-04-12
+full_link: 
+short_description: >
+  Uma pessoa que pode revisar e aprovar as contribuições do código Kubernetes.
+
+aka: 
+tags:
+- community
+---
+ Uma pessoa que pode revisar e aprovar as contribuições do código Kubernetes.
+
+<!--more--> 
+
+Embora a revisão de código esteja focada na qualidade e correção, a aprovação está focada na aceitação integral de uma contribuição. A aceitação inclui a compatibilidade anterior/posterior, adesão a API e convenções das flags, problemas sutis de desempenho e correção, interações com outras partes do sistema, etc. O status do aprovador tem escopo para uma parte código. Os aprovadores eram anteriormente referidos como mantenedores.
```

Comment:
Minor:
* Instead of "adesão a API e convenções das flags", "adesão a convenções de API e de flags" delivers the original meaning better.
* Suggestion: instead of "compatibilidade anterior/posterior", "retrocompatibilidade e compatibilidade futura" would sound better.